### PR TITLE
[Rule Tuning] Suspicious macOS MS Office Child Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1021

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by brokensound77 — APPROVED** [review]
feel free to add eql comments in the query for context if it helps

:shipit:

**Review by bm11100 — COMMENTED** [review]
We could add `Microsoft Outlook` as a parent as well and also look for these process names `"whoami", "hostname", "id", "uname", "ioreg"`.

**Comment by Samirbous** [comment]
> We could add `Microsoft Outlook` as a parent as well and also look for these process names `"whoami", "hostname", "id", "uname", "ioreg"`.

@bm11100 99% of the cases use Macro or exploit for office use Word, Excel, Powerpoint and won't trigger on Outlook (I'm not aware of any public technique where they use macro or alike on Outlook).  and for recon utilities as a child of office processes in general is less likely to happen first stage nor useful for prevention (its already noisy with sh and alike and I've excluded some of them).

File: rules/macos/initial_access_suspicious_mac_ms_office_child_process.toml [hunk 1]
```diff
@@ -47,7 +47,18 @@ process where event.type in ("start", "process_started") and
...
+ not process.args:
+        (
+          "ProductVersion",
+          "hw.model",
+          "ioreg",
+          "ProductName",
+          "ProductUserVisibleVersion",
+          "ProductBuildVersion",
+          "/Library/Application Support/Microsoft/MERP*/Microsoft Error Reporting.app/Contents/MacOS/Microsoft Error Reporting"
+        )
```
Comment (threat-punter) [inline rules/macos/initial_access_suspicious_mac_ms_office_child_process.toml:61]:
```suggestion
  ) and
  /* noisy false positives related to product version discovery and office errors reporting */
  not process.args:
    (
      "ProductVersion",
      "hw.model",
      "ioreg",
      "ProductName",
      "ProductUserVisibleVersion",
      "ProductBuildVersion",
      "/Library/Application Support/Microsoft/MERP*/Microsoft Error Reporting.app/Contents/MacOS/Microsoft Error Reporting"
    )
```

**Review by threat-punter — APPROVED** [review]

**Pull request merged**